packages/react-components/package.json

```diff
@@ -10,36 +10,55 @@
     "build:watch": "tsc --build --watch",
     "build:storybook": "tsc --build stories && ../../node_modules/.bin/build-storybook -s stories",
     "clean": "tsc --build --clean",
-    "test": "jest --watch",
-    "test:coverage": "jest --ci --coverage",
+    "test": "karma start karma.conf.js --singleRun",
```

Comment:
Maybe basic question, how does coverage work for watch mode? Does it patch the coverage data every run?

Should we keep the previous impl by having "test" be watch mode, unless the "CI" env is set, which it then change to single run mode? Or pass the "--singleRun" arg separately? aka `npm run test -- --singleRun`.

Reply:
> Maybe basic question, how does coverage work for watch mode? Does it patch the coverage data every run?

Yeah! I'll separate the coverage

> Should we keep the previous impl by having "test" be watch mode, unless the "CI" env is set, which it then change to single run mode?

This makes more sense. I'll do that. Thanks